core/textile/client.go

```diff
@@ -162,12 +162,20 @@ func (tc *textileClient) Start(ctx context.Context, cfg config.Config) error {
 	// Create key pair if not present
 	kc := keychain.New(tc.store)
 	log.Debug("Starting Textile Client: Generating key pair...")
+	// space app keys
 	if _, _, err := kc.GenerateKeyPair(); err != nil {
 		log.Debug("Starting Textile Client: Error generating key pair, key might already exist")
 		log.Debug(err.Error())
 		// Not returning err since it can error if keys already exist
 	}
 
+	// root signing keys
+	if _, _, err := kc.GenerateRootKeyPair(); err != nil {
+		log.Debug("Starting Textile Client: Error generating key pair, key might already exist")
```

Comment:
Maybe it would better if we actually check the error for an existing key error. So we can catch any other possible error that might have occurred and return that one.

Reply:
Yeah I thought the same too looking at all the existing places we do this skipped to make this PR quick. I agree will use this PR to clean it up as it's in a few places.